src/ert/_c_wrappers/enkf/model_config.py

```diff
@@ -1,299 +1,122 @@
 from ert._c_wrappers.enkf.time_map import TimeMap
 from ert._c_wrappers.sched import HistorySourceEnum
-from ert._c_wrappers.util import PathFormat
 
+logger = logging.getLogger(__name__)
 
-class ModelConfig(BaseCClass):
-    TYPE_NAME = "model_config"
 
-    _alloc = ResPrototype(
-        "void*  model_config_alloc(config_content, \
-                                   char*, \
-                                   ecl_sum)",
-        bind=False,
-    )
-    _alloc_full = ResPrototype(
-        "void*  model_config_alloc_full(int, \
-                                        int, \
-                                        char*, \
-                                        char*, \
-                                        char*, \
-                                        char*, \
-                                        char*, \
-                                        time_map, \
-                                        char*, \
-                                        history_source_enum, \
-                                        ecl_sum)",
-        bind=False,
-    )
-    _free = ResPrototype("void  model_config_free( model_config )")
-    _get_max_internal_submit = ResPrototype(
-        "int   model_config_get_max_internal_submit(model_config)"
-    )
-    _get_runpath_as_char = ResPrototype(
-        "char* model_config_get_runpath_as_char(model_config)"
-    )
-    _select_runpath = ResPrototype(
-        "bool  model_config_select_runpath(model_config, char*)"
-    )
-    _set_runpath = ResPrototype("void  model_config_set_runpath(model_config, char*)")
-    _get_enspath = ResPrototype("char* model_config_get_enspath(model_config)")
-    _get_history_source = ResPrototype(
-        "history_source_enum model_config_get_history_source(model_config)"
-    )
-    _select_history = ResPrototype(
-        "bool  model_config_select_history(model_config, history_source_enum, ecl_sum)"
-    )
-    _gen_kw_export_name = ResPrototype(
-        "char* model_config_get_gen_kw_export_name(model_config)"
-    )
-    _runpath_requires_iterations = ResPrototype(
-        "bool  model_config_runpath_requires_iter(model_config)"
-    )
-    _get_jobname_fmt = ResPrototype("char* model_config_get_jobname_fmt(model_config)")
-    _get_runpath_fmt = ResPrototype(
-        "path_fmt_ref model_config_get_runpath_fmt(model_config)"
-    )
-    _get_num_realizations = ResPrototype(
-        "int model_config_get_num_realizations(model_config)"
-    )
-    _get_obs_config_file = ResPrototype(
-        "char* model_config_get_obs_config_file(model_config)"
-    )
-    _get_data_root = ResPrototype("char* model_config_get_data_root(model_config)")
-    _get_time_map = ResPrototype(
-        "time_map_ref model_config_get_external_time_map(model_config)"
-    )
-    _get_last_history_restart = ResPrototype(
-        "int model_config_get_last_history_restart(model_config)"
-    )
+class ModelConfig:
+    TYPE_NAME = "model_config"
```

Comment:
Is thins still needed?

Reply:
no, it is not (methinks). thank you, good catch!

Reply:
fixed.